src/hooks/useGetOrders.tsx

```diff
@@ -149,6 +152,43 @@ export function useGetTxOrders(txHash: string): GetTxOrdersResult {
   return { orders, error, isLoading: isLoading || areErc20Loading, errorTxPresentInNetworkId }
 }
 
+export function useTxOrderExplorerLink(
+  txHash: string,
+  orders: Order[] | undefined,
+): ExplorerLinkProps | Record<string, unknown> {
+  const networkId = useNetworkId() || undefined
+  const [explorerLink, setExplorerLink] = useState<ExplorerLinkProps | Record<string, unknown>>({})
+
+  useEffect(() => {
+    if (!networkId) {
+      return
+    }
+
+    if (!orders?.length) {
+      for (const network of NETWORK_ID_SEARCH_LIST) {
+        //update provider to find tx in network
+        updateWeb3Provider(web3, network)
+        web3.eth.getTransaction(txHash).then((tx) => {
+          if (tx) {
+            setExplorerLink({
+              type: 'tx',
+              networkId: network,
+              identifier: txHash,
+              showLogo: true,
```

Comment:
I'm not convinced on showing Etherscan logo for blockscout, although not strongly against it
![Screen Shot 2022-02-23 at 12 46 07](https://user-images.githubusercontent.com/43217/155405405-7c6ee19c-aba4-4297-aa6f-a001064de3b3.png)

Reply:
> I'm not convinced on showing Etherscan logo for blockscout, although not strongly against it ![Screen Shot 2022-02-23 at 12 46 07](https://user-images.githubusercontent.com/43217/155405405-7c6ee19c-aba4-4297-aa6f-a001064de3b3.png)

Hey @alfetopito , then, we need to remove this logo from the footer links, and transaction/user address from the eponymous pages
![image](https://user-images.githubusercontent.com/70885163/155520792-ae64a1c4-19fc-44f2-93ff-fb26542a755b.png)

And again, we use Etherscan icons in the Cowswap for GC
![image](https://user-images.githubusercontent.com/70885163/155520933-a6d59ab2-f1f7-4316-8fbc-6d9666da5c8b.png)